taccsite_cms/secrets.example.py

```diff
@@ -0,0 +1,38 @@
+########################
+# DJANGO
+########################
+
+SECRET_KEY = 'CHANGE_ME'
+
+########################
+# STORAGE
+########################
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'PORT': '5432',
+        'NAME': 'taccsite',
+        'USER': 'postgresadmin',
+        'PASSWORD': 'taccforever',
+        'HOST': 'core_cms_postgres'
+    }
+}
+
+########################
+# SEARCH
+########################
+
+ES_AUTH = 'username:password'
+ES_HOSTS = 'http://elasticsearch:9200'
+ES_INDEX_PREFIX = 'cms-dev-{}'
+ES_DOMAIN = 'http://localhost:8000'
+
+HAYSTACK_CONNECTIONS = {
```

Comment:
@taoteg, should **all** of these stay in `secrets.py`?

- The `ES_AUTH` value seems secret.
- The `ES_…` values are expected by `HAYSTACK_CONNECTIONS`.

Reply:
Yes, keep in secrets. While only the `ES_AUTH` value is truly sensitive, it is easier to keep them all together for tracking purposes (and avoid leaking any seemingly harmless info about internal portal configs). 

We might should have another review as to "what values live in which files" again, it has been a while AFAIK since we evaluated the specific location of values distributed across the defaults/custom/local/secrets files.

Reply:
Moving `ES_…` values would break ES if `HAYSTACK…` is in any of those files.

I've hit loading order complexity before. I solved it via [new section in `settings.py` **after** settings/secrets import](https://github.com/TACC/Core-CMS/blob/v4.15.2/taccsite_cms/settings.py#L808-L814).

I think the complexity of `ES_` in many files is "not worth it". I feel safe to do so, but I defer to you.[^1]

[^1]: Would you feel compelled to update such files on servers or in Core-Portal-Deployments?

Reply:
I 100% agree that it is "not worth it" due to the second-order effects.

> Would you feel compelled to update such files on servers or in Core-Portal-Deployments?

I don't think any `ES_` values live anywhere _except_ in the `secrets.py` files, so you have to update the file on the individual host (and add any changes to the Stache entry, naturally).

Reply:
K. So, good to go this PR?

Reply:
my bad, yes - approved!